install/upgrade/update_to_dev.php

```diff
@@ -210,17 +211,29 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
          $tasksTable => [
             'FKEY' => [
                $tasksTable => 'plugin_flyvemdm_policies_id',
-               $table => 'id'
-            ]
+               $table      => 'id',
+            ],
          ],
-         $fleetTable => [
-            'FKEY' => [
+      ],
+      'LEFT JOIN'  => [
+         $fleetTable  => [
+            'itemtype_applied' => 'PluginFlyvemdmFleet',
```

Comment:
If I remember correctly, this use case if JOIN is not documented. I assumed this was not possible. If this works, maybe we should add a contribution to the developer documentation.

Reply:
The request is exacly how I wanted it to be. Nice.

Remains to find how to get the entities_id column from an agent without conflict with the entities_id from a fleet. Again, I remember that column aliasing is not supported by the query builder.

Getting the entiteis_id column from agents is needed to build the correct MQTT topic in the foreach() :/

Reply:
I added a new condition to agent join now the new query is this....

```SQL
SELECT
  `glpi_plugin_flyvemdm_policies`.`symbol`,
  `glpi_plugin_flyvemdm_tasks`.`id`,
  `glpi_plugin_flyvemdm_tasks`.`itemtype_applied`,
  `glpi_plugin_flyvemdm_tasks`.`items_id_applied`,
  `glpi_plugin_flyvemdm_fleets`.`entities_id`
FROM `glpi_plugin_flyvemdm_policies`
  INNER JOIN `glpi_plugin_flyvemdm_tasks`
    ON (`glpi_plugin_flyvemdm_tasks`.`plugin_flyvemdm_policies_id` = `glpi_plugin_flyvemdm_policies`.`id`)
  LEFT JOIN `glpi_plugin_flyvemdm_fleets` ON (`itemtype_applied` = 'PluginFlyvemdmFleet' AND
                                              `glpi_plugin_flyvemdm_tasks`.`items_id_applied` =
                                              `glpi_plugin_flyvemdm_fleets`.`id`)
  LEFT JOIN `glpi_plugin_flyvemdm_agents` ON (`itemtype_applied` = 'PluginFlyvemdmAgent' AND
                                              `glpi_plugin_flyvemdm_tasks`.`items_id_applied` =
                                              `glpi_plugin_flyvemdm_agents`.`id` AND
                                              (`glpi_plugin_flyvemdm_agents`.`plugin_flyvemdm_fleets_id` =
                                               `glpi_plugin_flyvemdm_fleets`.`id`))
WHERE `symbol` IN
      ('disableFmRadio', 'disableVoiceMail', 'disableCallAutoAnswer', 'disableVoiceDictation', 'disableUsbOnTheGo',
       'resetPassword', 'inventoryFrequency', 'disableSmsMms', 'disableStreamVoiceCall', 'disableCreateVpnProfiles')
```